doctr/travis.py

```diff
@@ -381,9 +381,9 @@ def commit_docs(*, added, removed):
 
     return False
 
-def push_docs():
+def push_docs(deploy_branch='gh-pages'):
     """
-    Push the changes to the ``gh-pages`` branch.
+    Push the changes to the branch named ``deploy_branch``.
```

Comment:
This sounds like the branch is named "deploy_branch". Not sure how to reword it better, though.